### PR TITLE
Moved Curriculm and Topic pool to tabs, added new Curriculum View

### PR DESCRIPTION
**Pull request by ChineduOzodi**
Creating a curriculum view that mirrors the curriculum view of old. Will soon add the ability to create a new curriculum that will then allow for drag and drop sub-topic addition from the topic pool.

### CODE REVIEW

File: src/app/services/subtopic.service.ts [hunk 1]
```diff
@@ -24,80 +24,80 @@ const HTTP_OPTIONS = {
...
-      subtopic, HTTP_OPTIONS);
+  deactivate(subTopic: SubTopic): Observable<Object> {
+    subTopic.name = this.deactivateName(subTopic.name);
+    return this.http.put(environment.apiUrl + `SubTopics/${subTopic.id}`,
+      subTopic, HTTP_OPTIONS);
   }
 
   /**
-   * The function used to reactivate a subtopic in theserver
+   * The function used to reactivate a SubTopic in theserver
```
Comment (MarcinSalamon) [inline src/app/services/subtopic.service.ts]: the server

> in theserver

**Review by MarcinSalamon — DISMISSED** [review]

File: src/app/routes/curriculum-editor/create-version/create-version.component.ts [hunk 2]
```diff
@@ -55,7 +55,8 @@ export class CreateVersionComponent {
       creatorId: JSON.parse(sessionStorage['user'])['id'],
       dateCreated: new Date(),
       numberOfWeeks: this.numberOfWeeks,
-      topics: this.selectedTopics
+      topics: this.selectedTopics,
+      curriculumWeeks: []
     };
     /*
      * After the Curriculum object is created, we make
```
Comment (danielsidhu) [inline src/app/routes/curriculum-editor/create-version/create-version.component.ts:62]: this. . . this is how you follow Git Style Guidlines

**Comment by danielsidhu** [comment]
I will require confirmation on when you are ready to merge. Other than that, the code and comments are excellent.

**Review by ChineduOzodi — COMMENTED** [review]
I looked over. It looks good but I can't approve it because I submitted it.

**Review by danielsidhu — APPROVED** [review]
aside from the conflicts resolved. it looks good

**Pull request merged**